Cannot open input file 'freetype.lib'

Trying to build on Windows (git bash) and getting:

```
$ cargo build
...
  = note: LINK : fatal error LNK1181: cannot open input file 'freetype.lib'
```

Here is what I think is the important part, the rest of the build above appears to be fine.

Does this help?

```
error: linking with `link.exe` failed: exit code: 1181
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "/NXCOMPAT" "/LIBPATH:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.0.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.1.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.10.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.11.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.12.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.13.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.14.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.15.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.2.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.3.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.4.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.5.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.6.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.7.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.8.rcgu.o" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.memory.9uwjrfiq-cgu.9.rcgu.o" "/OUT:C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.exe" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\memory.225fd3pg7bowbhs2.rcgu.o" "/OPT:REF,NOICF" "/DEBUG" "/NATVIS:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis" "/LIBPATH:C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps" "/LIBPATH:D:/pd/msys64/mingw64/lib" "/LIBPATH:C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librand-eb3f9ba04787da88.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librand_chacha-743891e211773de1.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libppv_lite86-f9315c508e59b2d6.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librand_core-8798460db946a36f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libgetrandom-0a689725e0a4f76c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsixtyfps-9e72773274c5a66e.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsixtyfps_rendering_backend_default-9d3a5f6597c3cc54.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsixtyfps_rendering_backend_gl-6779e8a90488f419.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libglutin-82d9abb464c78f1c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libglutin_wgl_sys-86c10ab306437012.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblibloading-e5c5f9a3c884a96c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libglutin_egl_sys-3490e6c6915ff9ac.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfontdb-08ea09ca9a179f4c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libmemmap2-7d5737072dea6215.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libresvg-47f68e2c5023106b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsvgfilters-2e7d0d1d604f04b0.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libtiny_skia-73195b4fdb240c88.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsafe_arch-496d89f98d28e730.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libarrayref-d4f3a51dfc6742ba.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libby_address-0d64b289a7045d81.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libusvg-e9bfe5e843eb338d.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libflate2-1da83e5c96c73658.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libminiz_oxide-115af0487ea98e28.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libadler-04c6bf50bd050034.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librctree-fbb528d464e6f340.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libbase64-f5b3aa0eb1893566.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsimplecss-0bc4816645911c23.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libroxmltree-2271265d244784f8.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libxmlparser-7a140740eeea5a29.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libdata_url-f9d7282eaadce276.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libkurbo-28fa0b7d97a7db7f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libxmlwriter-fda152a158f6a34b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsvgtypes-0c276306c2727aa6.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsiphasher-8918a378bbcc9760.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfloat_cmp-18b01d9c2d0a8505.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libcopypasta-d93837045306cfb3.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libclipboard_win-3dbc35166e07c604.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblazy_bytes_cast-b8d162b834ae1e2f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfont_kit-7502f42e29a66f15.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfloat_ord-555d61c3a47a55a3.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libwalkdir-722d2f049288cde5.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsame_file-d04dc94a56c3fd22.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libwinapi_util-d2175821ab23274d.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfreetype-48579905c9875fe1.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfreetype_sys-259f2d30263ff447.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libdwrote-ad6cfd62f5143db2.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libwio-00bb52edf9f667cb.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libpathfinder_geometry-002876d2da80d8d5.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libpathfinder_simd-7c94125cbb4719b7.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfemtovg-bb784bbff5ed4bd5.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librustybuzz-96d7e8ebe5f51e2a.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_bidi_mirroring-42115c9e820e94c6.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_script-3603922480c969fb.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_ccc-b8439514a8eac807.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_general_category-f00620c6114bfebb.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libglow-0b957cb0da6d6670.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libowned_ttf_parser-b9444666fba6c39c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libttf_parser-9e5628ff337704bb.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_segmentation-fb07b69a1dfa053a.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libunicode_bidi-1b4886de2f7d1c18.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libmatches-adeb056d84c852cc.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblru-061e28a0288bb312.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libgenerational_arena-bef649314b799805.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libcfg_if-bf39ef30cad20e5b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfnv-1cf62f355dbdf88b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\librgb-c4c3b9e9c8938743.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libimgref-ba2666f28cc1259d.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libimage-448982d87d22458e.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libcolor_quant-924dfd3ef842940c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libbytemuck-de23ea9485b95bf8.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libpng-7306d041a26f9f65.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libminiz_oxide-48bc76182bcc192b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libcrc32fast-5b032da2c5fff974.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libdeflate-574f407170dbca5e.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libbyteorder-3f27263a0cb1fb5f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libadler32-f67afbf676de8174.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libnum_rational-b91af8af9451e577.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libnum_iter-627142e082ce3d2c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libnum_integer-f8e02bd2f1196266.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libjpeg_decoder-217559ac73c874e3.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libwinit-8caff436c3a58436.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libraw_window_handle-3333527d107a2621.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblibc-ac9ae018c9c2c58b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libparking_lot-8bda7f6e648e2db1.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libparking_lot_core-785b66341fb74faf.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsmallvec-149cb8d8636b7980.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblock_api-32bdd0b90dffb2cc.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libbitflags-5a7f2fb1ad23847f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblog-48ce8c5c693b361d.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblazy_static-416bc711c514df92.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsixtyfps_rendering_backend_qt-66b3f4b04df3a34f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsixtyfps_corelib-4f523f4e306449a3.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libweak_table-54c43a42ecf3573e.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libpin_weak-e2b5670976d175b0.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libstrum-4a0767d847618bbb.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libslab-995b96797ce19b6b.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libonce_cell-ce1f8283b66b4db3.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblyon_algorithms-e463809b29049b70.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libsid-36cb975123caeffa.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libscopeguard-0634f7c415b19b08.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libscoped_tls_hkt-17c25bd1c756dc38.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libtriomphe-17ac6db76cb30286.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libstable_deref_trait-3faa3d662b2b9563.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libserde-4dbdaacf297d19b2.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblyon_path-2862191210a88ce9.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\liblyon_geom-0e35d2007822ed48.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libeuclid-128b7e48d5d92e0c.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libnum_traits-d684352c8941774f.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libarrayvec-990c204510618097.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libauto_enums-1bae0be925137bb1.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libinstant-e921677f6a8334b5.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libtime-9c86aa0de15b1870.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libtime_macros-a5e58fae202f6013.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libwinapi-3cd1ca0dfb97e3b2.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libstandback-33c8e3bdd399e972.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libcfg_if-ed41771e6a6a9eca.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libvtable-1305c44b5c73d0fe.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libconst_field_offset-ecfa4360ccf82886.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libfield_offset-99e2b4b3af4a8ce4.rlib" "C:\\Users\\cedri\\rust\\sixtyfps\\target\\debug\\deps\\libmemoffset-f9ba7d784f49d017.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-68b25ab3829cbbae.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_abort-c053daed2d2aeacf.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-a32547dedd895cf6.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libhashbrown-2d29ef3bbf523eaa.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_alloc-2a226fea424b96ba.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-e28afa8739a1fa1c.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcfg_if-5e97a47829d8661c.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-c7a4b246b1481113.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-2b5ec6322b49dd45.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-c1df47dd61fa6cd5.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-3cedc7817d8eb468.rlib" "C:\\Users\\cedri\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-c115f0a110b00510.rlib" "bcrypt.lib" "opengl32.lib" "freetype.lib" "advapi32.lib" "cfgmgr32.lib" "comctl32.lib" "d2d1.lib" "dwmapi.lib" "dwrite.lib" "dxgi.lib" "gdi32.lib" "imm32.lib" "kernel32.lib" "msimg32.lib" "ole32.lib" "opengl32.lib" "shcore.lib" "shell32.lib" "shlwapi.lib" "synchronization.lib" "user32.lib" "uxtheme.lib" "windowscodecs.lib" "winspool.lib" "advapi32.lib" "ws2_32.lib" "userenv.lib" "msvcrt.lib"
  = note: LINK : fatal error LNK1181: cannot open input file 'freetype.lib'
```

--- a/sixtyfps_runtime/rendering_backends/gl/Cargo.toml
+++ b/sixtyfps_runtime/rendering_backends/gl/Cargo.toml
@@ -50,13 +50,15 @@ ttf-parser = "0.12.0"
 winit = { version = "0.25", default-features = false }
 glutin = { version = "0.27", default-features = false }
 glow = { version = "0.10.0", default-features = false }
-font-kit = { version = "0.10", features = ["loader-freetype"] }
+font-kit = { version = "0.10", features = [] }
 fontdb = { version = "0.5.1", features = ["fs"] }
 
-# Require font-config from the system. Issue #88 indicates that the copy provided by servo-fontconfig may be incompatible
-# with distros at times.
 [target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios", target_arch = "wasm32")))'.dependencies]
+# Require font-config from the system on Linux. Issue #88 indicates that the copy provided by servo-fontconfig may be incompatible
+# with distros at times.
 servo-fontconfig = { version = "0.5", features = [ "force_system_lib" ] }
+# On Windows and macOS we don't need freetype (font-kit has coretext/directwrite backends)
+font-kit = { version = "0.10", features = ["loader-freetype"] }
 
 [target.'cfg(target_os = "macos")'.dependencies]
 cocoa = { version = "0.24.0" }

